src/users/users.service.ts

```diff
@@ -0,0 +1,34 @@
+import { Injectable } from '@nestjs/common';
+import {InjectRepository} from "@nestjs/typeorm";
+import {User} from "./users.entity";
+import {Repository} from "typeorm";
+
+@Injectable()
+export class UsersService {
+    constructor(@InjectRepository(User) private repo: Repository<User>) {}
+
+    getAll() {
+        return this.repo.find()
+    }
+
+    getById(id: number) {
+        return this.repo.findOneBy({id})
+    }
+
+    add(name: string) {
+        const newUser = this.repo.create({name})
+        return this.repo.save(newUser)
+    }
+
+    async remove(id: number) {
+        const user = await this.repo.findOneBy({id})
+        await this.repo.remove(user)
+    }
+
+    //TODO: Pomyśleć nad napisaniem generycznego mappera albo sprawdzenie czy nie ma libki lub type orm czegoś takiego nie posiada, zamiast ręcznie przypisywać pola.
```

Comment:
don't even know which part of this code can / should be auto-generated 😅 
but sure, if something can be automated, lets create a task in our project board! 👍🏻

Reply:
Now we have very simple example but usually `dto` object don't equal `entity` object 1 to 1. Especially when entities have such relations as one to many, many to many, etc. Thats why mappers are for. We will need to map `entity` to `dto` and `dto` to `entity`.

Reply:
got it! thanks for clarification! 🍻